.env.example

```diff
@@ -4,3 +4,4 @@ WS_MOPIDY_URL=<Mopidy Host Name e.g mopidy-staging.local>
 WS_MOPIDY_PORT=6680
 CLIENT_ID=<dev google client id>
 CLIENT_ID_PRODUCTION=<production google client id>
+NOW_PLAYING_URL=<now playing add track endpoint>
```

Comment:
I wonder if it's worth adding "Optional: now playing add track endpoint"

Reply:
Made clear it's optional & added a little bit to the README